plugins/modules/ovirt_host_network.py

```diff
@@ -264,6 +265,8 @@ def get_type_name(mode_number):
             'Load balance (balance-xor)',
             None,
             'Dynamic link aggregation (802.3ad)',
+            'Adaptive transmit load balancing (balance-tlb)',
+            None,
```

Comment:
Hmm, shouldn't we align with official doc here?

https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html/administration_guide/sect-network_bonding#Bonding_Modes

Reply:
@mnecas can you please help me to understand the purpose of these strings here?

Reply:
This just checks what is in API with what user added to ansible. 
Users can choose the mode as a number we add the strings and dictionaries to match with the API response.

Reply:
https://github.com/oVirt/ovirt-ansible-collection/pull/199/files#r551842364